lib/system/dyncalls.nim

```diff
@@ -53,6 +53,10 @@ proc procAddrError(name: cstring) {.noinline.} =
 # http://www.lua.org
 # mailto:info@lua.org
 
+# Note: would like to use a table, but can't since it's included in system.
+when defined(nimDebugDlOpen):
+  var libHandles: seq[(string,LibHandle)]
```

Comment:
You cannot GCed memory here since the GC can be loaded from a DLL...

Reply:
ok, removed that from this PR, will try to address this particular point in another PR